### PR TITLE
Possible way to paginate

### PR DESCRIPTION
**Pull request by minbi**
Not sure about requirements slice has on its arguments.

### CODE REVIEW

File: app/RecipeQueries.py [hunk 1]
```diff
@@ -4,11 +4,12 @@
 from enums import Cuisine
 import QueryHelpers
 
-def getAllRecipes(limit=10, **kwargs):
+def getAllRecipes(limit=10, page=1, **kwargs):
     """Gets all the recipes in the database, limiting the results to up to 'limit' results.
```
Comment (Joshie196) [inline app/RecipeQueries.py]: I'd update the description here to include page (also the description for limit is now incorrect)

File: app/RecipeQueries.py [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
 
     Args:
-        limit (Optional(int)): The uppoer-bound on the number of recipes to return. Defaults to 10. Must be positive.
+        limit (Optional(int)): The upper-bound on the number of recipes to return. Defaults to 10. Must be positive.
```
Comment (Joshie196) [inline app/RecipeQueries.py]: This isn't true anymore

**Comment by Joshie196** [comment]
Also don't forget to paginate ingredients!

File: app/RecipeQueries.py (hunk 2 above)
Comment (minbi) [inline app/RecipeQueries.py]: This is still true in a way?

Comment (Joshie196) [inline app/RecipeQueries.py]: You're right.

**Comment by Joshie196** [comment]
ship it!

**Pull request merged**